sql/instance_list.sql

```diff
@@ -77,7 +77,7 @@ INSERT INTO `instance_list` VALUES (0,'TEST',0,0,0.000,0.000,0.000,0,-1,-1,-1,-1
 -- INSERT INTO `instance_list` VALUES (29,'operation:black_pearl',61,30,0.000,0.000,0.000,0,-1,-1,-1,-1);
 -- INSERT INTO `instance_list` VALUES (30,'better_than_one',61,30,0.000,0.000,0.000,0,-1,-1,-1,-1);
 -- INSERT INTO `instance_list` VALUES (31,'seagull_grounded',79,30,-350,-15.245,380,0,-1,-1,-1,-1);
--- INSERT INTO `instance_list` VALUES (32,'requiem',79,30,0.000,0.000,0.000,0,-1,-1,-1,-1);
+-- INSERT INTO `instance_list` VALUES (32,'requiem',79,30,-458.409,-10.000,-320.966,90,-1,-1,-1,-1);
```

Comment:
This line is still commented out. Is that intentional?

Reply:
Im not sure if the team wants them open until all complete. The npcs the start the missions are also commented out so you can even pick the mission unless you remove the --

Reply:
if we want to open im game with removing the -- think this is the 5th or 6th assault ive made so far. There is at least 1 per zone working